bottomnavigation/bottomnavigation/src/main/java/com/microsoft/device/dualscreen/bottomnavigation/BottomNavigationView.kt

```diff
@@ -169,26 +215,30 @@ open class SurfaceDuoBottomNavigationView @JvmOverloads constructor(
         }
     }
 
-    private fun setScreenParameters(screenInfo: ScreenInfo) {
-        screenInfo.getHinge()?.let {
+    private fun setScreenParameters(foldingFeature: FoldingFeature) {
+        foldingFeature.bounds.let {
```

Comment:
I think this `let` is not needed here. I think `bounds` can not be null (an in fact you are not using the `?` before)